lib/diagnostic.rb

```diff
@@ -100,13 +104,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr.pop'
 
 # What would `arr` equal after:
 arr[arr.length + 2] = 99
 # Write your response as a ruby string
 
-Diagnostic.array_add_to = 'your answer here'
+Diagnostic.array_add_to = 'arr = [12, 34, 56, 67, nil, 99]'
```

Comment:
Missing a `nil` nice it would be adding 2 index's when you add the 99 into the original arr.

`Diagnostic.array_add_to = 'arr = [12, 34, 56, 67, nil, nil, 99]'`